pyznap/utils.py

```diff
@@ -85,7 +85,9 @@ def read_config(path):
 
     config = []
     options = ['key', 'frequent', 'hourly', 'daily', 'weekly', 'monthly', 'yearly', 'snap', 'clean',
-               'dest', 'dest_keys', 'compress', 'exclude', 'raw_send']
+               'dest', 'dest_keys', 'compress', 'exclude', 'raw_send', 'resume', 'dest_auto_create',
+               'retry', 'retry_interval', 'ServerAliveInterval', 'ServerAliveCountMax',
```

Comment:
`retry` is the number of retries.
`retry_interval` is the seconds between two retries
`ServerAliveInterval x ServerAliveCountMax` is the max time ssh wait if server have no response. We may set default value of them. (10,3) may be a good choice. `ssh` will sent heart beats every 10s and if get no response for 3 times, it will exit with error.

The default settings of `ServerAliveInterval` and  `ServerAliveCountMax` in a unconfigured `ssh` will make it freeze almost forever when the network is lost (ssh just wait for server for very long time and raise no error)
So we need the ssh to quit in time and retry the transfer